eks-without-vpc/variables.tf

```diff
@@ -0,0 +1,82 @@
+#
+# Variables Configuration
+#
+
+variable "cluster-name" {
+  type    = "string"
+}
+
+variable "ec2-instance-type" {
+  type    = "string"
```

Comment:
Please provide default value for instance type: `m5.xlarge`

Reply:
Done